lib/huginn_acumen_product_agent/concerns/acumen_product_query_concern.rb

```diff
@@ -126,6 +135,7 @@ def parse_product_request(products)
                 'acumenAttributes' => {
                     'info_alpha_1' => field_value(p, 'Inv_Product.Info_Alpha_1'),
                     'info_boolean_1' => field_value(p, 'Inv_Product.Info_Boolean_1'),
+                    'subtitle' => field_value(p, 'Inv_Product.SubTitle'),
```

Comment:
I think this should go in `Product.disambiguatingDescription`.
https://schema.org/disambiguatingDescription

Reply:
@Ken-Michalak Makes sense to me. Fixed